apps/video/index.html

```diff
@@ -151,18 +172,20 @@ <h1 id="video-title"></h1>
     <!-- This element is the template of each thumbnail item -->
     <div id="thumbnail-group-header" class="hidden">
       <!--
-        <li>
-          <div class="thumbnail-group-header"></div>
-          <ul class="thumbnail-group-container"></ul>
+        <li role="presentation">
+          <div class="thumbnail-group-header" role="heading"
+               aria-level="1"></div>
+          <ul class="thumbnail-group-container" role="listbox"
```

Comment:
I'm curious why 'aria-level' is used here in the thumbnail-group-header template but not in the thumbnail template below.